core/base/ftmTree/FTMTreeUtils.cpp

```diff
@@ -0,0 +1,162 @@
+#include <FTMTreeUtils.h>
+
+using namespace ttk;
+using namespace ftm;
+
+// --------------------
+// Is
+// --------------------
+
```

Comment:
put all functions within the ftm namespace.

Reply:
Done!